packages/asana/src/Adaptor.js

```diff
@@ -1,9 +1,6 @@
-import {
-  execute as commonExecute,
-  composeNextState,
-  expandReferences,
-  http,
-} from '@openfn/language-common';
+import { execute as commonExecute } from '@openfn/language-common';
+import { expandReferences } from '@openfn/language-common/util';
+import { request as commonRequest } from './Utils';
```

Comment:
This shouldn't be called `commonRequest` - it's very confusing! Because it's NOT common.request.

Maybe call it `requestHelper`?